docs/src/content/docs/showcase.mdx

```diff
@@ -18,11 +18,34 @@ Starlight is already being used in production. These are some of the sites aroun
 
 See all the [public project repos using Starlight on GitHub](https://github.com/withastro/starlight/network/dependents).
 
-## Community plugins
+## Plugins
+
+Starlight [plugins](/reference/plugins/) can customize Starlight configuration, UI, and behavior, while also being easy to share and reuse.
+This section lists the official plugins supported by the Starlight team and also community plugins that are maintained by Starlight users.
+
+### Algolia DocSearch
```

Comment:
I don't know if we are planning to support more official plugins in the future but I was thinking that maybe we could have an official plugins section and list the algolia plugin under that section.

I think this will help level the importance of official and community plugins which will encourage more people to share their creations.

Reply:
Yeah, I think you're totally right. I updated the PR to add an "Official plugins" section and if we want to go back to a structure closer to the [suggested one](https://github.com/withastro/starlight/pull/942/commits/d3ef3114e88b6b59d91d1c5e94435ec43b7213cc), we can easily revert or tweak it.